src/backend/core/tests/teams/test_core_api_teams_retrieve.py

```diff
@@ -80,4 +80,6 @@ def test_api_teams_retrieve_authenticated_related():
         "name": team.name,
         "slug": team.slug,
         "abilities": team.get_abilities(user),
+        "created_at": team.created_at.isoformat()[:-6] + "Z",
```

Comment:
```suggestion
        "created_at": team.created_at.isoformat().replace("+00:00", "Z"),
```

Reply:
nit
[c673178](https://github.com/numerique-gouv/people/pull/55/commits/c6731783cbc1880d16ce708e6638bf3aeb621d42)

Reply:
Actually is more explicit than a `[:-6]`, for the future myself 🤗

Reply:
(a reference the [Zen of Python](https://peps.python.org/pep-0020/#the-zen-of-python) "Explicit is better than implicit.")